homeassistant/components/systemmonitor/sensor.py

```diff
@@ -60,62 +66,188 @@
+    ),
+    "network_out": SysMonitorSensorEntityDescription(
+        key="network_out",
+        name="Network out",
+        native_unit_of_measurement=DATA_MEBIBYTES,
+        icon="mdi:server-network",
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+        mandatory_arg=True,
+    ),
+    "packets_in": SysMonitorSensorEntityDescription(
+        key="packets_in",
+        name="Packets in",
+        native_unit_of_measurement=" ",
+        icon="mdi:server-network",
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+        mandatory_arg=True,
+    ),
+    "packets_out": SysMonitorSensorEntityDescription(
+        key="packets_out",
+        name="Packets out",
+        native_unit_of_measurement=" ",
+        icon="mdi:server-network",
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+        mandatory_arg=True,
     ),
-    "ipv4_address": ("IPv4 address", "", "mdi:server-network", None, True),
-    "ipv6_address": ("IPv6 address", "", "mdi:server-network", None, True),
-    "last_boot": ("Last boot", None, None, DEVICE_CLASS_TIMESTAMP, False),
-    "load_15m": ("Load (15m)", " ", CPU_ICON, None, False),
-    "load_1m": ("Load (1m)", " ", CPU_ICON, None, False),
-    "load_5m": ("Load (5m)", " ", CPU_ICON, None, False),
-    "memory_free": ("Memory free", DATA_MEBIBYTES, "mdi:memory", None, False),
-    "memory_use": ("Memory use", DATA_MEBIBYTES, "mdi:memory", None, False),
-    "memory_use_percent": (
-        "Memory use (percent)",
-        PERCENTAGE,
-        "mdi:memory",
-        None,
-        False,
+    "throughput_network_in": SysMonitorSensorEntityDescription(
+        key="throughput_network_in",
+        name="Network throughput in",
+        native_unit_of_measurement=DATA_RATE_MEGABYTES_PER_SECOND,
+        icon="mdi:server-network",
+        state_class=STATE_CLASS_TOTAL,
+        mandatory_arg=True,
     ),
-    "network_in": ("Network in", DATA_MEBIBYTES, "mdi:server-network", None, True),
-    "network_out": ("Network out", DATA_MEBIBYTES, "mdi:server-network", None, True),
-    "packets_in": ("Packets in", " ", "mdi:server-network", None, True),
-    "packets_out": ("Packets out", " ", "mdi:server-network", None, True),
-    "throughput_network_in": (
-        "Network throughput in",
-        DATA_RATE_MEGABYTES_PER_SECOND,
-        "mdi:server-network",
-        None,
-        True,
+    "throughput_network_out": SysMonitorSensorEntityDescription(
+        key="throughput_network_out",
+        name="Network throughput out",
+        native_unit_of_measurement=DATA_RATE_MEGABYTES_PER_SECOND,
+        icon="mdi:server-network",
+        state_class=STATE_CLASS_TOTAL,
+        mandatory_arg=True,
     ),
-    "throughput_network_out": (
-        "Network throughput out",
-        DATA_RATE_MEGABYTES_PER_SECOND,
-        "mdi:server-network",
-        None,
-        True,
+    "process": SysMonitorSensorEntityDescription(
+        key="process",
+        name="Process",
+        native_unit_of_measurement=" ",
```

Comment:
?

Reply:
I guess this was done as workaround to get this sensor considered as number ... would be a "#" as unit ok to imply that this is only an counting (_there is no real unit for amount of processes_)?

Reply:
Yes, define a state_class

Reply:
```suggestion
```

Reply:
Sometimes the solution is easier than expected 🙈